workload/scripts/workload-install.ps1

```diff
@@ -162,7 +162,13 @@ else
 }
 
 if ($DotnetTargetVersionBand -eq "<auto>") {
-    $DotnetTargetVersionBand = $DotnetVersionBand
```

Comment:
what do you think check the version ?

    if ($DotnetVersion[0] -eq "6")
    {
        $DotnetTargetVersionBand = $DotnetVersionBand
    }
    else
    {
        $IsPreviewVersion = $DotnetVersion.Contains("-preview") -or $DotnetVersion.Contains("-rc") -or $DotnetVersion.Contains("-alpha")
        if ($IsPreviewVersion -and ($SplitVersion.Count -ge 4)) {
            $DotnetTargetVersionBand = $DotnetVersionBand + $SplitVersion[2].SubString(3) + $VersionSplitSymbol + $($SplitVersion[3])
        }
        else {
            $DotnetTargetVersionBand = $DotnetVersionBand
        }
    }

Reply:
yes, true true. We don't want to diverse the install scripts! I will update. Thanks for reviewing.